config.toml

```diff
@@ -160,34 +160,34 @@ js = [
 fullversion = "v1.19.0"
 version = "v1.19"
 githubbranch = "v1.19.0"
-docsbranch = "release-1.19"
+docsbranch = "master"
 url = "https://kubernetes.io"
 
 [[params.versions]]
-fullversion = "v1.18.0"
+fullversion = "v1.18.8"
 version = "v1.18"
-githubbranch = "v1.18.0"
+githubbranch = "v1.18.8"
 docsbranch = "release-1.18"
-url = "https://v-18.docs.kubernetes.io"
+url = "https://v-1.18.docs.kubernetes.io"
```

Comment:
nit: v1-18.docs.kubernetes.io ?

Reply:
Hi @kbhawkey Thanks for your review. Fixed it :)